config.yaml

```diff
@@ -0,0 +1,3 @@
+sink: glog
+kubeconfig: on
+metric-prefix: kube
```

Comment:
Dendronaut uses `.dendronaut.yaml`. Is this a similar type of configuration, and if so should we be consistent?

That said I don't have a preference on either style.

Reply:
This was the style from the upstream repo. All I did was change it to yaml, since everything in kubernetes is yaml, it didnt make any sense to suddenly have json configurations.